infra/base-images/base-builder/compile_centipede

```diff
@@ -28,9 +28,9 @@ cp "$BIN_DIR/libcentipede_runner.pic.a" "$LIB_FUZZING_ENGINE"
 export DFTRACING_FLAGS='-fsanitize-coverage=trace-loads'
 export CENTIPEDE_FLAGS=`cat "$SRC/centipede/clang-flags.txt" | tr '\n' ' '`
 export LIBRARIES_FLAGS="-ldl -lrt -lpthread $SRC/centipede/weak.o"
-export CC='/clang/bin/clang'
-export CCC='/clang/bin/clang++'
-export CXX='/clang/bin/clang++'
+export CC='/usr/local/bin/clang'
```

Comment:
No need to define these at all. Let's just remove these lines because they're already defined by default: https://github.com/google/oss-fuzz/blob/1b63c5b4d2f1e179639d7e684fd15c88a168e907/infra/base-images/base-clang/Dockerfile#L51